### PR TITLE
remove update_id in UpdateBuilder

### PR DESCRIPTION
**Pull request by MarinPostma**
Removing the `update_id` from `UpdateBuidler`, since it serves no purpose. I had introduced it when working in HA some time ago, but I think there are better ways to do it now, so it can be removed an stop being in our way.

### CODE REVIEW

**Comment by ManyTheFish** [comment]
Wait for [the PR on error message](https://github.com/meilisearch/milli/pull/412) to be merged, before merging this one.
Thanks! 🙏

File: milli/src/search/facet/filter_condition.rs [hunk 1]
```diff
@@ -0,0 +1,929 @@
...
+        .unwrap();
+        let expected = FilterCondition::from_str(
+            &rtxn,
+            &index,
+            "channel = gotaga AND (timestamp = 44 OR channel != ponce)",
+        )
+        .unwrap();
+        assert_eq!(condition, expected);
+    }
+}
```
Comment (ManyTheFish) [inline milli/src/search/facet/filter_condition.rs:929]: This file is not imported in milli, you may not want to add it to this PR.
Moreover, it seems unrelated.

**Review by ManyTheFish — CHANGES_REQUESTED** [review]

File: milli/src/search/facet/filter_condition.rs (hunk 1 above)
Comment (MarinPostma) [reply milli/src/search/facet/filter_condition.rs:929]: uhm the rebase must have picked it up

**Review by ManyTheFish — APPROVED** [review]
bors merge
